jenkins-slaves/jenkins-slave-golang/Dockerfile

```diff
@@ -16,10 +16,9 @@ RUN if [ -z $GO_VERSION ] ; then GO_VERSION=${GO_VERSION_DEFAULT} ; fi && \
     mkdir -p /usr/src/go/src/redhat && \
     tar -xzf /usr/go${GO_VERSION}.linux-amd64.tar.gz && \
     mv $(pwd)/go /usr/local/ && \
-    go get -u github.com/golang/dep/cmd/dep && \
     chown -R 1001 /usr/src/go && \
     chown -R 1001 /usr/local/go && \
-    chown -R 1001 ${HOME}/.cache/go-build && \
+    chmod g+w /usr/src/go && \
```

Comment:
whats `/usr/src/go` used for?

Reply:
It can be, for sure. This was the path used before this PR, but typically GOPATH is `${HOME}/go`